packages/api/src/env.js

```diff
@@ -49,4 +58,22 @@ export function envAll (_, env, event) {
   const clusterAuthToken = env.CLUSTER_BASIC_AUTH_TOKEN || (typeof CLUSTER_BASIC_AUTH_TOKEN === 'undefined' ? undefined : CLUSTER_BASIC_AUTH_TOKEN)
   const headers = clusterAuthToken ? { Authorization: `Basic ${clusterAuthToken}` } : {}
   env.cluster = new Cluster(env.CLUSTER_API_URL || CLUSTER_API_URL, { headers })
+
+  try {
+    if ((env.S3_ACCESS_KEY_ID || S3_ACCESS_KEY_ID) && (env.S3_SECRET_ACCESS_KEY_ID || S3_SECRET_ACCESS_KEY_ID)) {
+      const s3Endpoint = env.S3_BUCKET_ENDPOINT || (typeof S3_BUCKET_ENDPOINT === 'undefined' ? undefined : S3_BUCKET_ENDPOINT)
+      env.s3Client = new S3Client({
+        endpoint: s3Endpoint,
+        forcePathStyle: !!s3Endpoint, // Force path if endpoint provided
```

Comment:
only way we can enforce a path for testing. In alternative, given backup is not required, we can remove this and not test with it, as we should be able to trust `@aws-sdk/client-s3` module